rustfmt-core/rustfmt-lib/tests/target/issue-3550.rs

```diff
@@ -0,0 +1,39 @@
+fn main() {
+    assert(
+        long()
+            .long()
+            .long()
+            .long()
+            .long()
+            .long()
+            .long()
+            .long()
+            .long()
+            .long()
+            .long()
+            .long()
+            .long()
+            .long()
+            .long()
+            .long(),
+    );
+
+    assert!(
+        long()
+            .long()
+            .long()
+            .long()
+            .long()
+            .long()
+            .long()
+            .long()
+            .long()
+            .long()
+            .long()
+            .long()
+            .long()
+            .long()
+            .long()
+            .long(),
+    );
```

Comment:
What's this second one validating that's not covered by the first one?

Reply:
The first is a function, the second is the assert! macro. Per #3550 we want to see that assert! is formatted similar to regular function calls.

Reply:
ha! and that's why reviewing PRs when half asleep isn't a great idea :smile: